users/migrations/0013_auto_20171101_2338.py

```diff
@@ -0,0 +1,33 @@
+# -*- coding: utf-8 -*-
+# Generated by Django 1.11.5 on 2017-11-01 23:38
+from __future__ import unicode_literals
+
+from django.db import migrations, models
+
+
+class Migration(migrations.Migration):
+
+    dependencies = [
+        ('users', '0012_usergroup'),
+    ]
+
+    operations = [
```

Comment:
Since we never have deployed this code and ran these migrations on production, we can safely squash them into one migration and delete both `0012_usergroup.py` and `0013_auto_20171101_2338.py`. You may find https://docs.djangoproject.com/en/1.11/topics/migrations/#migration-squashing useful (sorry if you're already familiar with squashing migrations)